qt-webkit@2.3.rb

```diff
@@ -0,0 +1,30 @@
+class QtWebkitAT23 < Formula
+  desc "Qt port of WebKit (insecure, you shouldn't use it)"
```

Comment:
Drive-by comment: people will use it. If you don't think they should: don't package it or you're at least partially responsible for any security issues they find themselves affected by.